Dockerfile

```diff
@@ -27,23 +26,23 @@ RUN mkdir /root/.ssh && \
     chmod 700 /root/.ssh/ && \
     umask 0077 && echo "$GIT_SSH_KEY" >/root/.ssh/id_rsa && \
     eval "$(ssh-agent -s)" && ssh-add /root/.ssh/id_rsa
+
+# Install npm
+RUN apt-get update && apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update && apt-get install -y nodejs npm
+
 RUN echo "installing npm packages"
 RUN npm install
 
 RUN echo "Starting the before_script section" && \
 		dart --version
 
-# Use pub from Dart 2 to initially resolve dependencies since it is much more efficient.
-COPY --from=dart2 /usr/lib/dart /usr/lib/dart2
-RUN echo "Running Dart 2 pub get.." && \
-	_PUB_TEST_SDK_VERSION=1.24.3 timeout 5m /usr/lib/dart2/bin/pub get --no-precompile
-
 RUN pub get && \
 		echo "before_script section completed"
 RUN echo "Starting the script section" && \
 		./tool/check_version.sh && \
 		echo "script section completed"
```

Comment:
These chaining of commands is leftover from the auto-conversion from smithy.yml. Additionally, Rosie has its own function for ensuring that the version is bumped and its enabled for this repo. We should be able to consolidate all of this to:

```Dockerfile
RUN dart --version
RUN pub get
```

Reply:
How can one check if Rosies's version check is enabled?